optional_plugins/html/setup.py

```diff
@@ -24,7 +24,7 @@
       url='http://avocado-framework.github.io/',
       packages=find_packages(),
       include_package_data=True,
-      install_requires=['avocado-framework', 'jinja2'],
+      install_requires=['avocado-framework', 'jinja2<3.0.0'],
```

Comment:
@richtja we also have  `python3-jinja2` in `python-avocado.spec`. Should we also set the version there or not? (not sure)

Reply:
@richtja looks like `python3-jinja2` inside `python-avocado.spec` is for `rpm` packages. Not sure if Fedora or RHEL will upgrade this package too.

Reply:
@beraldoleal Fedora and EPEL should *not* upgrade with new *major* versions, so I don't expect this update to be required.